### DISCUSSION TITLE
2008/04/02/labview%E6%96%B0%E7%89%B9%E6%80%A7%E9%A2%84%E8%A7%88%EF%BC%9A%E5%88%A9%E7%94%A8%E8%B4%9D%E5%A1%9E%E5%B0%94%E6%9B%B2%E7%BA%BF%E6%8F%90%E9%AB%98%E7%A8%8B%E5%BA%8F%E8%BF%90%E8%A1%8C%E6%95%88%E7%8E%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#213] [Announcements]
# 2008/04/02/labview%E6%96%B0%E7%89%B9%E6%80%A7%E9%A2%84%E8%A7%88%EF%BC%9A%E5%88%A9%E7%94%A8%E8%B4%9D%E5%A1%9E%E5%B0%94%E6%9B%B2%E7%BA%BF%E6%8F%90%E9%AB%98%E7%A8%8B%E5%BA%8F%E8%BF%90%E8%A1%8C%E6%95%88%E7%8E%87

目前 LabVIEW 在程序框图上的连线或横或竖，没有斜向连的，更不会出现曲线。如果需要连接的两个端点不在同一高度上，则数据连线会出现数个90度的直角弯。这种连线方式虽然整洁，但却会严重影响 LabVIEW 的运行速度。

https://blog.qizhen.xyz/2008/04/02/labview%E6%96%B0%E7%89%B9%E6%80%A7%E9%A2%84%E8%A7%88%EF%BC%9A%E5%88%A9%E7%94%A8%E8%B4%9D%E5%A1%9E%E5%B0%94%E6%9B%B2%E7%BA%BF%E6%8F%90%E9%AB%98%E7%A8%8B%E5%BA%8F%E8%BF%90%E8%A1%8C%E6%95%88%E7%8E%87

**Comment by ruanqizhen** [comment]
Qizhen
2008年04月8日 21:55 编辑
人工荨麻疹没好，但也不严重。我也不吃药，随他去了。至于痛不痛苦，不在于疾病本身，而在于你的心态。比人工荨麻疹惨的病有的是，人家患者还是活得好好的。所以，如果这个病治愈不了，就改变自己的心情吧。

回复
泡
2008年04月8日 17:11 编辑
朋友,你好!
你的荨麻疹好了吗?你现在还在吃那个盐酸西替利嗪吗?
那以后要是结婚生孩子怎么办呀,生孩子好像不能吃药的哦?
我也是人工荨麻疹哦,痛苦死喽?
有空加QQ聊聊410938350

回复
Zhen
2008年04月2日 20:08 编辑
汗一个~~
偶今天才看到，差点就信了

回复
杰
2008年04月2日 19:17 编辑
好像我是比较理性一点，这两者本来就是风马牛的，
和楼下的几位比，好像少了不少幽默细胞，呵呵。

回复
arrowpig
2008年04月2日 11:37 编辑
这不是挺cool的嘛！

回复
atppp
2008年04月2日 10:16 编辑
faint 我差一点就相信了！！！

回复
永青
2008年04月2日 09:32 编辑
有点意思

回复
